src/components/HtmlView.js

```diff
@@ -53,12 +53,12 @@ const htmlConfig = {
   ignoredTags: IGNORED_TAGS
 };
 
-export const HtmlView = (props) => (
+export const HtmlView = ({ html, tagsStyles, openWebScreen }) => (
   <HTML
-    {...props}
+    html={html}
     {...htmlConfig}
-    onLinkPress={(evt, href) => openLink(href)}
-    tagsStyles={{ ...styles.html, ...props.tagsStyles }}
+    onLinkPress={(evt, href) => openLink(href, openWebScreen)}
+    tagsStyles={{ ...styles.html, ...tagsStyles }}
```

Comment:
I can see that openWebScreen method is applied everywhere is needed but where did you define what this method/function is doing ?  why is it not part the commits ?

Reply:
commented here: https://github.com/ikuseiGmbH/smart-village-app-app/pull/84#discussion_r473092530